### PR TITLE
Upgrade Loki to 2.7.2

### PR DESCRIPTION
**Pull request by aminesnow**
This PR upgrades Loki from 2.6.1 to 2.7.2

### CODE REVIEW

File: configuration/components/loki.libsonnet [hunk 1]
```diff
@@ -294,6 +301,7 @@ local defaults = {
       per_stream_rate_limit: '3MB',
       per_stream_rate_limit_burst: '15MB',
       split_queries_by_interval: '30m',
+      deletion_mode: 'disabled',
     },
     querier: {
```
Comment (periklis) [inline configuration/components/loki.libsonnet:306]: According to upgrade docs (https://grafana.com/docs/loki/latest/upgrading/#engine-query-timeout-is-deprecated) we should drop `query.engine.timeout` in favor of `query.query_timeout`. This is probably a hindsight from the last update to 2.7.0

**Review by periklis — CHANGES_REQUESTED** [review]

File: configuration/components/loki.libsonnet (hunk 1 above)
Comment (aminesnow) [reply configuration/components/loki.libsonnet:306]: I saw that. Thing is, I don't know which value to keep. WDYT the value should be?

Comment (periklis) [reply configuration/components/loki.libsonnet:306]: we should keep the `query.query_timeout` with the value as is.

**Review by periklis — COMMENTED** [review]
Looks good to me

**Review by periklis — APPROVED** [review]
Approving this as per upcoming v2.7.3 is not going to provide changes for the loki component jsonnet here.

**Pull request merged**